earl/src/main/java/com/einmalfel/earl/RSSItem.java

```diff
@@ -181,6 +195,10 @@ public String getDescription() {
     return null;
   }
 
+  public String getContentEncoded() {
```

Comment:
Do we need this method at all instead of usual final field access? I understand that it has some (not awful lot of) value, but why not to add getItunesAuthor() then? and also getItunesDuration(), getMediaRating(), getMediaKeywods() e.t.c